File: Modul-4/Soal-4/PRAK404-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main() {
    int pilihan = 0;
    float pertama, kedua;

    while (pilihan != 5) {
        printf("Pilih program \n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukan pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan > 5 || pilihan < 1) {
            printf("Input anda salah, silahkan coba lagi\n"); }
        else if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Azri");
            break; } 
        else {
            printf("Masukan nilai pertama: ");
            scanf(" %f", &pertama);

            printf("Masukan nilai kedua: ");
            scanf(" %f", &kedua);}
        
        switch (pilihan)
        {
        case 1:
            printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, pertama+kedua);
            break;
        case 2:
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, pertama-kedua);
            break;
        case 3:
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, pertama*kedua);
            break;
        case 4:
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", pertama, kedua, pertama/kedua);
            break;
        }
        printf("\n");}
        }

File: Modul-4/Soal-4/PRAK404-2410817110016-MuhammadAzriRahman.py
pilihan = 0

while pilihan != 5:
        print("Pilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")

        pilihan = int(input("Masukan pilihan: "))

        if pilihan > 5 or pilihan < 1:
            print("Input anda salah, silahkan coba lagi\n")
            continue
        elif pilihan == 5:
            print("Terimakasih, telah menggunakan kalkulator Azri")
            break

        pertama = float(input("Masukan nilai pertama: "))
        kedua = float(input("Masukan nilai kedua: "))

        if pilihan == 1:
            print(f"Hasil penjumlahan antara {pertama:.2f} dengan {kedua:.2f} adalah {pertama + kedua:.2f}")
        elif pilihan == 2:
            print(f"Hasil pengurangan antara {pertama:.2f} dengan {kedua:.2f} adalah {pertama - kedua:.2f}")
        elif pilihan == 3:
            print(f"Hasil perkalian antara {pertama:.2f} dengan {kedua:.2f} adalah {pertama * kedua:.2f}")
        elif pilihan == 4:
                print(f"Hasil pembagian antara {pertama:.2f} dengan {kedua:.2f} adalah {pertama / kedua:.2f}")
        print(" ")
